docs/changes.rst

```diff
@@ -155,13 +155,19 @@ Compatibility notes:
 - Repositories in the "authenticated" mode are now treated as the unencrypted
   repositories they are.
 
+- The client-side repository cache now holds unencrypted data for better speed.
 
 New features:
 
-- integrity checking for important files used by borg:
+- reimplemented the RepositoryCache, size-limited caching of decrypted repo
+  contents, integrity checked via xxh64.
+- integrity checking using xxh64 for important files used by borg:
 
   - repository: index and hints files
   - cache: chunks and files caches, archive.chunks.d
+- compact chunks.archive.d
```

Comment:
Maybe "Reduced space usage of chunks.archive.d. Existing caches are migrated during a cache sync".